### PR TITLE
Feat : Added Signup Page

### LINKED ISSUE #11: Update : signUp page
Added links in signUp form

### PR DESCRIPTION
**Pull request by RahulSingh9131**
1. signup Page
signupPage

### CODE REVIEW

File: css/screens/signupPage.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/* css for signUp page */
```
Comment (ImrozNezami) [inline css/screens/signupPage.css:1]: Remove comment.

File: css/screens/signupPage.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    position: fixed;
+    padding: 2rem;
+    top: 13%;
+    left: 0;
+    right: 0;
+    max-width: 70vw;
+}
+.signup-section .signup-container form{
+    background-color: var(--white-color);
+    border-radius: .5rem;
```
Comment (ImrozNezami) [inline css/screens/signupPage.css]: You can use px for values less than 1rem.

Comment (RahulSingh9131) [reply css/screens/signupPage.css]: sure...

File: css/screens/signupPage.css (hunk 1 above)
Comment (RahulSingh9131) [reply css/screens/signupPage.css:1]: Actually many suggested me to add comments before writing css for better understanding.

File: docs/signUp/signup.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login | ShoeStore</title>
+    <link rel="stylesheet" href="../../css/main.css">
```
Comment (sadathshariff) [inline docs/signUp/signup.html]: you can keep the main CSS file at the end since you'll be overriding the styles

Comment (RahulSingh9131) [reply docs/signUp/signup.html]: oh sure ..thanks .

**Review by sadathshariff — APPROVED** [review]

**Review by rahulcs754 — APPROVED** [review]
looks all good. 
make sure you change your suggestion

**Review by rohit8686 — APPROVED** [review]
Everything looks good !

**Pull request merged**